ubiquity/testdata/ca.pem.metadata

```diff
@@ -0,0 +1,27 @@
+[
+{
+	"name":"Chrome Browser M39",
```

Comment:
do you want to test the current "real" metadata? The Firefox logic is slightly different from Chrome (chain vs. leaf) and is worth testing. The chain creation code in helpers should be able to help you dynamically create the test cases.